ui/src/apps/clusterInfo/components/ClusterInfoTable.js

```diff
@@ -38,9 +30,57 @@ function ComponentPanelTable({ cluster }) {
       key: 'deploy_path',
       ellipsis: true,
     },
+    {
+      title: t('cluster_info.component_table.status'),
+      dataIndex: 'status',
+      key: 'status',
+      width: 100,
+      render: (status, node) => {
+        if (!status) {
+          return
+        }
+        let statusBanner = <span>{t(`cluster_info.component_table.${status}`)} </span>;
```

Comment:
why not 

```js
<span>{t(`cluster_info.component_table.${status}`)} { tidb ? ... : null }</span>;
```

HTML structure is simpler

Reply:
I think using `statusBanner` is OK, I don't like mixing them together...